<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSwiftData</class>
 <widget class="QMainWindow" name="CSwiftData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>swiftdata</string>
  </property>
  <widget class="QWidget" name="qw_CentralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="vl_CentralWidget">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="swift::gui::components::CInfoBarWebReadersStatusComponent" name="comp_InfoBar">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="sw_CentralWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="swift::gui::components::CDataMainInfoAreaComponent" name="comp_MainInfoArea"/>
      <widget class="QWidget" name="comp_EmptyPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mb_SwiftData">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_InfoAreas">
    <property name="title">
     <string>&amp;Info areas</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Mapping">
    <property name="title">
     <string>&amp;Mapping</string>
    </property>
    <addaction name="menu_MappingMaxData"/>
    <addaction name="menu_MappingMaxMapping"/>
   </widget>
   <widget class="QMenu" name="menu_Internals">
    <property name="title">
     <string>Internals</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_InfoAreas"/>
   <addaction name="menu_Help"/>
   <addaction name="menu_Mapping"/>
   <addaction name="menu_Internals"/>
  </widget>
  <widget class="QStatusBar" name="sb_SwiftData"/>
  <action name="menu_FileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="menu_FileSettingsDirectory">
   <property name="text">
    <string>Settings directory</string>
   </property>
  </action>
  <action name="menu_FileResetSettings">
   <property name="text">
    <string>Reset settings</string>
   </property>
  </action>
  <action name="menu_FileReloadStyleSheets">
   <property name="text">
    <string>Reload style sheets</string>
   </property>
  </action>
  <action name="menu_WindowFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="menu_WindowMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="menu_MappingMaxMapping">
   <property name="text">
    <string>Max. mapping area</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M, M</string>
   </property>
  </action>
  <action name="menu_MappingMaxData">
   <property name="text">
    <string>Max. data area</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M, D</string>
   </property>
  </action>
  <action name="menu_WindowToggleOnTop">
   <property name="text">
    <string>Toggle stay on top</string>
   </property>
  </action>
  <action name="actionNavigator">
   <property name="text">
    <string>Navigator</string>
   </property>
  </action>
  <action name="menu_DebugMetaTypes">
   <property name="text">
    <string>Debug metatypes (to console)</string>
   </property>
  </action>
  <action name="menu_JsonBootstrapTemplate">
   <property name="text">
    <string>JSON bootstrap template</string>
   </property>
  </action>
  <action name="menu_InternalsEnvVars">
   <property name="text">
    <string>Env.variables</string>
   </property>
  </action>
  <action name="menu_InternalsMetatypes">
   <property name="text">
    <string>Metatypes</string>
   </property>
  </action>
  <action name="menu_InternalsSetup">
   <property name="text">
    <string>Setup</string>
   </property>
  </action>
  <action name="menu_InternalsCompileInfo">
   <property name="text">
    <string>Compile info</string>
   </property>
  </action>
  <action name="menu_InternalsDisplayCachedFiles">
   <property name="text">
    <string>All cache files</string>
   </property>
  </action>
  <action name="menu_InternalsDeleteCachedFiles">
   <property name="text">
    <string>Delete cached fies</string>
   </property>
  </action>
  <action name="menu_SettingsDirectory">
   <property name="text">
    <string>Directory</string>
   </property>
  </action>
  <action name="menu_SettingsReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="menu_SettingsFiles">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="menu_CacheDirectory">
   <property name="text">
    <string>Directory</string>
   </property>
  </action>
  <action name="menu_CacheFiles">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="menu_CacheReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="menu_JsonDownloadTemplate">
   <property name="text">
    <string>JSON download template</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CDataMainInfoAreaComponent</class>
   <extends>QWidget</extends>
   <header>gui/components/datamaininfoareacomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CInfoBarWebReadersStatusComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/infobarwebreadersstatuscomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
